civiform/civiform
# Use USWDS alert for "Please select a file" error on file upload question

### Description

If a user is on a file upload question and clicks "Save & next" without uploading a file, we display an error saying "Please select a file". This PR updates that error to use the USWDS error alert UI component.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Created unit and/or browser tests which fail without the change (if possible) -- **Existing tests should already cover this**

#### User visible changes


### Instructions for manual testing

1. As an applicant, open a file upload question.
2. Don't upload a file, then click "Save&next".
3. Verify the USWDS error alert is shown. (I also verified that the alert is announced by my VoiceOver screen reader.)
4. Upload a file.
5. Verify the USWDS error is hidden as soon as your file is uploaded.
